openshift/e2e-common.sh

```diff
@@ -96,24 +73,39 @@ function install_serverless(){
 }
 
 function run_e2e_tests(){
+  local test_name="${1:-}"
+  local failed=0
+  local channels=messaging.knative.dev/v1alpha1:InMemoryChannel,messaging.knative.dev/v1alpha1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel
+  local common_opts="-channels=$channels --kubeconfig $KUBECONFIG --imagetemplate $TEST_IMAGE_TEMPLATE"
 
   header "Running tests with Single Tenant Channel Based Broker"
-  oc apply -f test/config/st-channel-broker.yaml || return 1
-  wait_until_pods_running $EVENTING_NAMESPACE || return 1
-  go_test_e2e -timeout=90m -parallel=12 ./test/e2e -brokerclass=ChannelBasedBroker -channels=messaging.knative.dev/v1alpha1:InMemoryChannel,messaging.knative.dev/v1alpha1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel \
-    --kubeconfig "$KUBECONFIG" \
-    --imagetemplate "$TEST_IMAGE_TEMPLATE" \
-    ${options} || failed=1
+
+  oc patch KnativeEventing knative-eventing \
```

Comment:
OK, this we can not yet do on master/release-next _and_ also not on 0.15.x  (problems w/ the serverless operator and running these branches)

Reply:
OK